rbd/mirror_peer_site.go

```diff
@@ -156,7 +156,7 @@ type MirrorPeerSite struct {
 	SiteName   string
 	MirrorUUID string
 	ClientName string
-	LastSeen   C.time_t
+	LastSeen   int
```

Comment:
I'm a bit iffy on the use of int in the case that the C type is larger than the go int type on the platform. In the other PR you ended up using int64, why did you use int here?

Reply:
I had the exact same thing in the mind while using int type here, and was expecting a comment like this 😅.
I used int64 in the other PR because Timespec's property have int64 as the defined type.

Anyway, I will update the type to int64 here as well.

Reply:
Since here this is not related to some unix-y low level API, I'm in favor of using time. Time here, else the user must find out how to interpret the int.

Reply:
Hmm, ok. If we have consensus on this, then let's close this PR.

Reply:
Ahh, my bad.
I didn't understood his comment earlier. Thanks for the clarification, I will update the PR accordingly.

Reply:
Oops, sorry, the space in `time. Time` was a confusing typo. Yes, I meant `time.Time`.

Reply:
No worries :)
I have updated the code to use `time.Time` as the type for `LastSeen` property.